Angular/Angular guidelines and best practices/Formatting, naming and best practices.md

```diff
@@ -1048,6 +1048,31 @@ Some notes:
 - We wrapped everything in one `ng-container` to avoid calling the pipe twice
 - Be careful in case that your pipe can return a valid falsy value, as in that case the `*ngIf` will not render the content; in such case you might consider using a [custom `*ngLet`](https://github.com/ngrx-utils/ngrx-utils#nglet-directive) structural directive
 
+## Auto unwrap default exports when lazy loading
+
+With Angular v15 it is possible to leverage default exports to shorten the syntax when lazy loading module or a standalone component. Note that the CLI still generates modules and components without the default export so you will need to add that by yourself.
+
+```typescript
+@Component({...})
+export default class LazyCmp { ... }
+
+// Without default export
+{
+  path: 'lazy',
+  loadComponent: () => import('./lazy-file').then(m => m.LazyCmp),
```

Comment:
lazy-file is weird naming, just use index/detail whatever, just something that reflects its a component that represents a page. There is nothing like a "lazy" component